packages/web-shared/components/VideoPlayer/VideoPlayer.js

```diff
@@ -242,9 +244,9 @@ function VideoPlayer(props = {}) {
     ? props.parentNode?.stream?.sources[0]?.uri
     : videoMedia?.sources[0]?.uri;
 
-  const config = useMemo(()=>{
-    if (source.includes("youtube.com")){
-      return undefined
+  const config = useMemo(() => {
+    if (source.includes('youtube.com')) {
```

Comment:
this will still have the issue of `source` being undefined and throwing an error. that was the original issue that took berean down. needs to be `source?.includes(`

Reply:
fixed